src/components/date-picker/date-picker.tsx

```diff
@@ -110,9 +110,13 @@ export class DatePicker {
 
     public componentWillLoad() {
         this.useNative = isIOSDevice() || isAndroidDevice();
-        this.formattedValue = this.dateFormatter.formatDateByType(
+        this.format = this.format
+            ? this.format
+            : this.dateFormatter.getDateFormat(this.type);
```

Comment:
Nice. But I don't quite get how this works since `this.format` is an `@Prop()`-decorated property, and those aren't supposed to be possible to change from inside the component without configuring them to be mutable… 🤔

I'm going to need to update this for the native-picker stuff anyway, so leave as-is, and I might update to make use of `@State() internalFormat` or something.

Reply:
You're right about that! Sounds like a good plan. Could it be that it's possible to set just because it's not sent in as a prop?

Reply:
Maybe… 🤷‍♂️